# avoid duplicate definition of enum plist_format_t introduced in libplist 2.3.0
# origin: https://github.com/archlinux/svntogit-packages/blob/packages/libimobiledevice/trunk/libplist-2.3.0.diff
# 
diff --git i/common/utils.c w/common/utils.c
index 7f66ec2..7ebe997 100644
--- i/common/utils.c
+++ w/common/utils.c
@@ -314,7 +314,7 @@ int plist_read_from_filename(plist_t *plist, const char *filename)
 	return 1;
 }
 
-int plist_write_to_filename(plist_t plist, const char *filename, enum plist_format_t format)
+int plist_write_to_filename(plist_t plist, const char *filename, plist_format_t format)
 {
 	char *buffer = NULL;
 	uint32_t length;
diff --git i/common/utils.h w/common/utils.h
index 2c3acec..4faa5bd 100644
--- i/common/utils.h
+++ w/common/utils.h
@@ -51,13 +51,8 @@ char *generate_uuid(void);
 void buffer_read_from_filename(const char *filename, char **buffer, uint64_t *length);
 void buffer_write_to_filename(const char *filename, const char *buffer, uint64_t length);
 
-enum plist_format_t {
-	PLIST_FORMAT_XML,
-	PLIST_FORMAT_BINARY
-};
-
 int plist_read_from_filename(plist_t *plist, const char *filename);
-int plist_write_to_filename(plist_t plist, const char *filename, enum plist_format_t format);
+int plist_write_to_filename(plist_t plist, const char *filename, plist_format_t format);
 
 void plist_print_to_stream(plist_t plist, FILE* stream);
 
